modules/custom/cu_js_includes/pages/cu_js_includes.confirm_form.inc

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Need to improve comments, a lot are cut and paste from the tracking pixel module